<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>413</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>poppins</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Schedule</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font-family: poppins;
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="font">
      <font>
       <family>poppins</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="lblName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Employee Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Bio No.:</string>
      </property>
     </widget>
     <widget class="QLabel" name="bioNumTxt">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblBio_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Time In:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBio_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Time Out:</string>
      </property>
     </widget>
     <widget class="QLabel" name="timeInTxt">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="timeOutTxt">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="holidayNameTxt">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblType">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Holiday Name:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateOfWork">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblType_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date of Work:</string>
      </property>
     </widget>
     <widget class="QLabel" name="empNameTxt">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hours Worked:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hoursWorkedTxt">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QLabel" name="lblBio_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Schedule:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblType_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Type of Day:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>311</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblType_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Validation Note:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>311</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">	background-color: &quot;#344273&quot;;
	color: white;</string>
      </property>
      <property name="text">
       <string> Update</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../MainFrame/Resources/Icons/update.svg</normaloff>../../MainFrame/Resources/Icons/update.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="typeOfDayCb">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Ordinary Day</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Regular Holiday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Special Holiday</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="costCenterFrom">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>99</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>00:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23:00</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="costCenterFrom_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>99</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>00:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23:00</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
